src/main/java/ru/javarush/quest/servlet/QuestServlet.java

```diff
@@ -0,0 +1,101 @@
+package ru.javarush.quest.servlet;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import ru.javarush.quest.entity.Answer;
+import ru.javarush.quest.entity.User;
+import ru.javarush.quest.exception.QuestServletException;
+import ru.javarush.quest.exception.QuestUnknownException;
+import ru.javarush.quest.repository.QuestRepository;
+import ru.javarush.quest.repository.UserRepository;
+import ru.javarush.quest.service.QuestService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
+@WebServlet("/quest")
+@Slf4j
+@Getter
+public class QuestServlet extends HttpServlet {
+    private QuestRepository questRepository;
+    private UserRepository userRepository;
+    private QuestService questService;
+
+    @Override
+    public void init() {
+        try {
+            questService = new QuestService();
```

Comment:
On controller level we should use service, not repository

Reply:
Fixed